regulations/static/regulations/js/source/views/main/main-view.js

```diff
@@ -249,6 +251,24 @@ var MainView = Backbone.View.extend({
 
     setHandlers: function() {
         this.applyTablePlugin();
+        this.applyClipboardPlugin();
+    },
+
+    applyClipboardPlugin: function() {
+        // Create anchor tag for copy to clipboard
+        this.$el.find('.poster').before(
+           $('<a>', {
+                'class': 'clipboard-link',
+                text: 'Copy this text to your clipboard',
+                title: 'Copy this text to your clipboard',
+                href: '#'
+            })
+        );
+        var links = new Clipboard('.clipboard-link', {
+            target: function(trigger) {
+                return trigger.nextElementSibling;
+            }
+        });
```

Comment:
Minor, but if you are going to make the other edits, consider this too:

This proceeds by 1) creating several links with certain characteristics then 2) adding functionality to those links. Would it make more sense to just do this inside the first loop? That way it wouldn't require a global selector. Something like:
```javascript
this.$el.find('.whatever-selector').each(function(_, copyable) {
    var anchor = $('<a>', ...)
    new Clipboard(anchor, {target: function(_) {return copyable;}});
    $(copyable).before(anchor);
});
```